filebeat/beater/crawler.go

```diff
@@ -15,58 +15,59 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package crawler
+package beater
 
 import (
 	"fmt"
 	"sync"
 
 	"github.com/elastic/beats/v7/filebeat/channel"
-	"github.com/elastic/beats/v7/filebeat/fileset"
 	"github.com/elastic/beats/v7/filebeat/input"
 	"github.com/elastic/beats/v7/filebeat/input/file"
 	"github.com/elastic/beats/v7/filebeat/registrar"
 	"github.com/elastic/beats/v7/libbeat/beat"
 	"github.com/elastic/beats/v7/libbeat/cfgfile"
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/logp"
-
-	_ "github.com/elastic/beats/v7/filebeat/include"
 )
 
-type Crawler struct {
+type crawler struct {
 	inputs          map[uint64]*input.Runner
 	inputConfigs    []*common.Config
 	out             channel.Factory
 	wg              sync.WaitGroup
-	InputsFactory   cfgfile.RunnerFactory
-	ModulesFactory  cfgfile.RunnerFactory
+	inputsFactory   cfgfile.RunnerFactory
+	modulesFactory  cfgfile.RunnerFactory
 	modulesReloader *cfgfile.Reloader
 	inputReloader   *cfgfile.Reloader
 	once            bool
```

Comment:
can `once` be changed to a more descriptive name? especially since there is already a package-scope variable with this name that is now shadowed by function parameters in the crawler helpers.

Reply:
once if set by the `-once` cli flag :)

I'd keep it for now

Reply:
You mean the package-level variable? I meant change the name of `crawler.once`, since it's being moved into a package that already uses that name (and I'm not sure what it's supposed to convey)... but the struct field doesn't rely on a particular external interface or flag as far as I can see.

Reply:
The CLI argument is defined by filebeat and passed as last argument to newCrawler.